rose/server/score.py

```diff
@@ -14,11 +14,13 @@ def process(players, track):
     # status, used for resolving collisions.
 
     for player in players.itervalues():
+        player_start_point = player.lane * config.cells_per_player
         if player.action == actions.LEFT:
-            if player.x > 0:
+            # the left player should be start on x=0, and right in the middle
+            if player.x > player_start_point:
                 player.x -= 1
         elif player.action == actions.RIGHT:
-            if player.x < config.matrix_width - 1:
+            if player.x < player_start_point + config.cells_per_player - 1:
                 player.x += 1
```

Comment:
i intend to disagree. this is the logic to have it works. without it the score means nothing